draft-ietf-tls-subcerts.md

```diff
@@ -142,17 +163,17 @@ mechanisms like proxy certificates {{RFC3820}} for several reasons:
 
 * There is no change needed to certificate validation at the PKI layer.
 * X.509 semantics are very rich.  This can cause unintended consequences if a
-  service owner creates a proxy cert where the properties differ from the leaf
-  certificate. For this reason, delegated credentials have very restricted
+  service owner creates a proxy certificate where the properties differ from the leaf
+  certificate.  For this reason, delegated credentials have very restricted
   semantics which should not conflict with X.509 semantics.
 * Proxy certificates rely on the certificate path building process to establish
   a binding between the proxy certificate and the server certificate.  Since
-  the cert path building process is not cryptographically protected, it is
+  the certificate path building process is not cryptographically protected, it is
   possible that a proxy certificate could be bound to another certificate with
   the same public key, with different X.509 parameters.  Delegated credentials,
   which rely on a cryptographic binding between the entire certificate and the
   delegated credential, cannot.
-* Delegated credentials are bound to specific versions of TLS. This prevents
+* Delegated credentials are bound to specific versions of TLS.  This prevents
```

Comment:
maybe add they are also bound to a specific signature algorithm

Reply:
Done.